src/actions/userActions.js

```diff
@@ -57,4 +71,24 @@ export function login(creds) {
         console.log('Login!', err); // eslint-disable-line
       });
   };
-}
\ No newline at end of file
+}
+
+export function changePassword(creds) {
+  var player = storeUtil.getPlayer(creds.playerId);
+  var playerName = player ? player.alias : 'John Doe';
+
+  return dispatch => {
+    return http.post('/users/ChangePassword', creds)
+      .then(function(data) {
+        if (!data) {
+          dispatch(passwordChangedFailed(playerName));
+        } else {
+          dispatch(passwordChanged(playerName));
+          dispatch(initialLoad());
```

Comment:
bij het inloggen wordt de initialLoad opnieuw gedaan om zo de Spelers.ContactDetails en de Matchen.Opstelling ook beschikbaar te hebben. 
Voor andere acties (zoals deze changePassword) is dat niet nodig..